src/ucs/vfs/base/vfs_obj.c

```diff
@@ -181,7 +181,7 @@ static ucs_vfs_node_t *ucs_vfs_node_add(void *parent_obj,
     }
 
     /* generate the relative path */
-    ucs_snprintf_zero(rel_path_buf, sizeof(rel_path_buf), rel_path, ap);
+    vsnprintf(rel_path_buf, sizeof(rel_path_buf), rel_path, ap);
```

Comment:
what is wrong with ucs_snprintf_zero?

Reply:
It was just the wrong usage of ucs_snprintf_zero. Because the method has signature for another use case.